### PR TITLE
multipart blob upload

### PR DESCRIPTION
**Pull request by dehann**

### CODE REVIEW

File: src/services/BlobStore.jl [hunk 1]
```diff
@@ -134,6 +137,42 @@ end
...
+
+    response = GQL.execute(
+        client,
+        GQL_COMPLETEUPLOAD;
+        variables = Dict(
+            "blobId" => blobId, 
+            "completedUpload" => cui, 
+        ),
+        throw_on_execution_error = true,
+    )
```
Comment (Affie) [inline src/services/BlobStore.jl:171]: I'm slowly transitioning to the convenience wrapper `executeQql`.

File: src/services/BlobStore.jl [hunk 2]
```diff
@@ -134,6 +137,42 @@ end
...
+
+    return response.data["completeUpload"]
```
Comment (Affie) [inline src/services/BlobStore.jl:173]: You should be able to use `handleQuery` here.

File: src/services/BlobStore.jl (hunk 1 above)
Comment (dehann) [reply src/services/BlobStore.jl:171]: did not find easy example so just going with copy from single part upload

File: src/services/BlobStore.jl (hunk 2 above)
Comment (dehann) [reply src/services/BlobStore.jl:173]: same

**Comment by dehann** [comment]
Hi @Affie , I tested up and down looks to be working.

File: src/services/BlobStore.jl [hunk 3]
```diff
@@ -152,6 +191,74 @@ end
 
 ##
 
+
+function addBlob!(
+    store::NavAbilityBlobStore,
+    filepath::AbstractString,
+    blobId::UUID = uuid4();
+    chunkSize::Integer = UPLOAD_CHUNK_SIZE_HASH,
+)
```
Comment (Affie) [inline src/services/BlobStore.jl:200]: We should rather call this `uploadFile` and have an `addBlob!` that matches the api. 
I haven't had a use case for `uploadFile` yet, but it's good to have as I can see some uses.

**Pull request merged**